test/integration/targets/copy/tasks/src_directory_contaning_one_single_file.yml

```diff
@@ -0,0 +1,59 @@
+# Test copying a source directory that contains only a single file
```

Comment:
This first test case succeeds with or without the patch, and it doesn't add missing test coverage, so I think it should be removed.